app/index.js

```diff
@@ -43,30 +43,24 @@ module.exports = generators.Base.extend({
 
     var readmeTpl = _.template(this.fs.read(this.templatePath('README.md')));
 
-    this.composeWith('node:app', {
-      options: {
-        babel: false,
-        boilerplate: false,
-        name: this.props.name,
-        projectRoot: 'generators',
-        skipInstall: this.options.skipInstall,
-        readme: readmeTpl({
-          generatorName: this.props.name,
-          yoName: this.props.name.replace('generator-', '')
-        })
-      }
-    }, {
-      local: require('generator-node').app
+    this.composeWith(require.resolve('generator-node/generators/app'), {
+      babel: false,
+      boilerplate: false,
+      name: this.props.name,
+      projectRoot: 'generators',
+      skipInstall: this.options.skipInstall,
+      readme: readmeTpl({
+        generatorName: this.props.name,
+        yoName: this.props.name.replace('generator-', '')
+      })
     });
 
-    this.composeWith('generator:subgenerator', {
+    this.composeWith(require.resolve('../subgenerator'), {
       arguments: ['app']
```

Comment:
This needs to be the name of the argument you want to pass in. It's treated the same as an option now.

`{namespace: 'app'}`